### ISSUE TITLE
VideoOSD

### ISSUE THREAD

**Issue by aleksslab** [b-jesch/skin.estuary.modv2#53]
If you put ActivateWindow(TVChannels) instead of ActivateWindow(PVROSDChannels) (for 70040), would it be more informative. Or not?

**Comment by b-jesch** [comment]
Yes, there is some work in progress for simplifying and uniforming some of the PVR menues. Be patient ;) .

**Comment by aleksslab** [comment]
Ok I will wait

**Comment by aleksslab** [comment]
Hi. MB you need some help?

**Comment by b-jesch** [comment]
Hi, I have a full-time job and family. ;)  Sometimes I also need a little time off. But I will push the latest changes into the Kodinerds repo today.

**Comment by aleksslab** [comment]
"I have a full-time job and family" me too:). I mean help with "...simplifying and uniforming some of the PVR menues."
